uberfire-extensions/uberfire-layout-editor/uberfire-layout-editor-client/src/main/java/org/uberfire/ext/layout/editor/client/components/columns/ComponentColumn.java

```diff
@@ -414,6 +431,11 @@ public void onSelected() {
             }
         }
     }
+    
+    @Override
+    public List<LayoutEditorElementPart> getLayoutEditorElementParts() {
+        return parts;
+    }
 
     public interface View extends UberElement<ComponentColumn> {
 
```

Comment:
I'd add a @PreDestroy to clear the LayoutColumnParts generated with the ManagedInstance

Reply:
@pefernan @jesuino `ManagedInstance` keeps track of the create instances and destroy them automatically when the bean instance in which it is injected is destroyed, so no need for `@PreDestroy` here.

Reply:
@paulovmr good to know that, I always clear the ManagedInstances with  @Predestroys... so @jesuino forget this comment